api_checkout.md

```diff
@@ -121,10 +120,13 @@ The Shop Information captures the details of the essential for the checkout.
 |-------------------|----------|---------|------------------------------------------------------------------------------------------------------------|
 | `sku`             | `string` |         | SKU of the product                                                                                         |
 | `amount`          | `int`    |         | Number of products / packages                                                                              |
-| `weight`          | `int`    | 0       | Weight of product in case of of a weighable (a not packaged) product                                       |
+| `weight`          | `int`    | 0       | Weight of product in case of a weighable (not packaged) product                                            |
+| `weightUnit`      | `string` |         | Unit for weighable item to calculate price                                                                 |
+| `referenceUnit`   | `string` |         | Reference unit for weighable item, e.g. if weightUnit is g referenceUnit is kg                             |
 | `units`           | `int`    | 0       | Number of units in a package in case of bundle or piece product                                            |
-| `price`           | `int`    |         | Price of the product in case of an encoded price                                                           |
+| `price`           | `int`    |         | Price of the product for single unit, piece, weightUnit, etc                                               |
 | `priceOrigin`     | `string` |         | Origin of the price (`master` price from the products list, `client` the price was provided by the client) |
+| `totalPrice`      | `int`    |         | totalPrice of the product, e.g. amount * price or weight * price                                           |
```

Comment:
Besser "Total price of the Line Item". Die Berechnung stimmt so nicht ganz und ich würde diese entfernen.